sys/kern/device.c

```diff
@@ -91,3 +91,14 @@ resource_t *device_take_resource(device_t *dev, res_type_t type, int rid,
 
   return rle->res;
 }
+
+device_t *device_if_find_impl(device_t *dev, size_t iface,
```

Comment:
The `iface` argument should be of the `drv_if_t` type.

Reply:
Wait, we have that??? Ok then, I'll fix it soon!

Reply:
Did that.